SignalServiceKit/src/Devices/OWSBlockedPhoneNumbersMessage.m

```diff
@@ -32,14 +32,19 @@ - (instancetype)initWithPhoneNumbers:(NSArray<NSString *> *)phoneNumbers
     return self;
 }
 
-- (OWSSignalServiceProtosSyncMessageBuilder *)syncMessageBuilder
+- (nullable SSKProtoSyncMessageBuilder *)syncMessageBuilder
 {
-    OWSSignalServiceProtosSyncMessageBlockedBuilder *blockedPhoneNumbersBuilder =
-        [OWSSignalServiceProtosSyncMessageBlockedBuilder new];
-    [blockedPhoneNumbersBuilder setNumbersArray:_phoneNumbers];
-    OWSSignalServiceProtosSyncMessageBuilder *syncMessageBuilder = [OWSSignalServiceProtosSyncMessageBuilder new];
-    [syncMessageBuilder setBlocked:[blockedPhoneNumbersBuilder build]];
+    SSKProtoSyncMessageBlockedBuilder *blockedBuilder = [SSKProtoSyncMessageBlockedBuilder new];
+    [blockedBuilder setNumbers:_phoneNumbers];
+    NSError *error;
+    SSKProtoSyncMessageBlocked *_Nullable blockedProto = [blockedBuilder buildAndReturnError:&error];
+    if (error || !blockedProto) {
+        OWSFail(@"%@ could not build protobuf: %@", self.logTag, error);
+        return nil;
+    }
 
+    SSKProtoSyncMessageBuilder *syncMessageBuilder = [SSKProtoSyncMessageBuilder new];
+    [syncMessageBuilder setBlocked:blockedProto];
```

Comment:
The wrapper's `build()` and `serialize()` methods throw validation exceptions, so all of our proto construction logic now explicitly handles exceptions.  That's the bulk of this PR: incredibly repetitive changes around errors.